src/Plugins/Init.php

```diff
@@ -98,4 +106,68 @@ private function init(): void
 
         exit(0);
     }
+
+    private function initLaravel(): int
+    {
+        View::render('components.badge', [
+            'type' => 'INFO',
+            'content' => 'Laravel installation detected, pest-plugin-laravel will be installed.',
+        ]);
+
+        exec('composer require pestphp/pest-plugin-laravel 2.x-dev', result_code: $result);
```

Comment:
Can you check the code we are using on Jetstream / Breeze to run these sort of commands?